packages/lab/stories/progress/progress.linear.stories.tsx

```diff
@@ -79,84 +56,26 @@ export const All: ComponentStory<typeof LinearProgress> = () => (
       right: 0,
     }}
   >
-    <SaltProvider mode="light">
-      <LinearExamples />
-    </SaltProvider>
-    <SaltProvider mode="dark">
-      <LinearExamples />
-    </SaltProvider>
+    <LinearExamples />
   </div>
 );
 
 export const Default: ComponentStory<typeof LinearProgress> = () => (
-  <div>
-    <div>
-      <h3>size=small</h3>
-      <LinearProgress size="small" value={38} />
-    </div>
-    <div style={{ marginTop: 50 }}>
-      <h3>size=medium</h3>
-      <LinearProgress size="medium" value={38} />
-    </div>
-    <div style={{ marginTop: 50 }}>
-      <h3>size=large</h3>
-      <LinearProgress size="large" value={38} />
-    </div>
-  </div>
+  <LinearProgress value={38} />
 );
 
 export const Disabled: ComponentStory<typeof LinearProgress> = () => (
-  <div>
-    <div>
-      <h3>size=small</h3>
-      <LinearProgress disabled size="small" value={38} />
-    </div>
-    <div style={{ marginTop: 50 }}>
-      <h3>size=medium</h3>
-      <LinearProgress disabled size="medium" value={38} />
-    </div>
-    <div style={{ marginTop: 50 }}>
-      <h3>size=large</h3>
-      <LinearProgress disabled size="large" value={38} />
-    </div>
-  </div>
-);
-
-export const Indeterminate: ComponentStory<typeof LinearProgress> = () => (
-  <div>
-    <div>
-      <h3>size=small</h3>
-      <LinearProgress size="small" value={38} variant="indeterminate" />
-    </div>
-    <div style={{ marginTop: 50 }}>
-      <h3>size=medium</h3>
-      <LinearProgress size="medium" value={38} variant="indeterminate" />
-    </div>
-    <div style={{ marginTop: 50 }}>
-      <h3>size=large</h3>
-      <LinearProgress size="large" value={38} variant="indeterminate" />
-    </div>
-  </div>
+  <LinearProgress disabled value={38} />
 );
 
 export const ProgressingValue: ComponentStory<typeof LinearProgress> = () => (
   <ProgressWithControls ProgressComponent={LinearProgress} />
 );
 
-export const LinearQuery: ComponentStory<typeof LinearProgress> = () => (
+export const MaxValue: ComponentStory<typeof CircularProgress> = () => (
   <div>
-    <div>
-      <h3>size=small</h3>
-      <LinearProgress size="small" value={38} variant="query" />
-    </div>
-    <div style={{ marginTop: 50 }}>
-      <h3>size=medium</h3>
-      <LinearProgress size="medium" value={38} variant="query" />
-    </div>
-    <div style={{ marginTop: 50 }}>
-      <h3>size=large</h3>
-      <LinearProgress size="large" value={38} variant="query" />
-    </div>
+    <h3> max = 500, value = 250</h3>
+    <LinearProgress aria-label="Download" value={250} max={500} />
```

Comment:
in the value description it says value should be between 0 and 100?

Reply:
will update description